sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -2284,6 +2284,128 @@ case class MakeDate(
     copy(year = newFirst, month = newSecond, day = newThird)
 }
 
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = "_FUNC_(year, month, day, hour, min, sec[, timezone]) - Create local date-time from year, month, day, hour, min, sec and timezone fields. ",
```

Comment:
Does the expression accept a time zone? I don't see that ...

Reply:
Updated. Sorry about that.